src/Entity/JobOffer.php

```diff
@@ -83,7 +86,7 @@ class JobOffer
     /**
      * @ORM\Column(type="string", length=14)
      * 
-     * @Groups({"admin"})
+     * @Groups({"admin", "detail", "list"})
```

Comment:
I don't see why we need to show the status in detail and list groups. The company that the job belongs to will see the status in the admin group. BTW, out of scope.